spring-cloud-gcp-starters/pom.xml

```diff
@@ -1,47 +1,49 @@
-		<module>spring-cloud-gcp-starter-firestore</module>
-		<module>spring-cloud-gcp-starter-trace</module>
-		<module>spring-cloud-gcp-starter-logging</module>
-		<module>spring-cloud-gcp-starter-metrics</module>
-		<module>spring-cloud-gcp-starter-sql-mysql</module>
-		<module>spring-cloud-gcp-starter-sql-postgresql</module>
-		<module>spring-cloud-gcp-starter-sql-mysql-r2dbc</module>
-		<module>spring-cloud-gcp-starter-sql-postgres-r2dbc</module>
-		<module>spring-cloud-gcp-starter-security-iap</module>
-		<module>spring-cloud-gcp-starter-vision</module>
-		<module>spring-cloud-gcp-starter-data-firestore</module>
-		<module>spring-cloud-gcp-starter-bigquery</module>
-		<module>spring-cloud-gcp-starter-security-firebase</module>
-		<module>spring-cloud-gcp-starter-secretmanager</module>
-		<module>spring-cloud-gcp-starter-kms</module>
-	</modules>
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>spring-cloud-gcp</artifactId>
+    <groupId>com.google.cloud</groupId>
+    <version>3.4.1-SNAPSHOT</version>
+  </parent>
+  <packaging>pom</packaging>
+  <name>Spring Framework on Google Cloud Starters</name>
+  <description>Spring Framework on Google Cloud Starters</description>
+  <artifactId>spring-cloud-gcp-starters</artifactId>
+  <organization>
+    <name>Google</name>
+    <url>https://cloud.google.com</url>
+  </organization>
+  <properties>
+    <main.basedir>${basedir}/..</main.basedir>
+  </properties>
+  <modules>
+    <module>spring-cloud-gcp-starter</module>
+    <module>spring-cloud-gcp-starter-bus-pubsub</module>
+    <module>spring-cloud-gcp-starter-cloudfoundry</module>
+    <module>spring-cloud-gcp-starter-pubsub</module>
+    <module>spring-cloud-gcp-starter-config</module>
+    <module>spring-cloud-gcp-starter-storage</module>
+    <module>spring-cloud-gcp-starter-data-spanner</module>
+    <module>spring-cloud-gcp-starter-data-datastore</module>
+    <module>spring-cloud-gcp-starter-firestore</module>
+    <module>spring-cloud-gcp-starter-trace</module>
+    <module>spring-cloud-gcp-starter-logging</module>
+    <module>spring-cloud-gcp-starter-metrics</module>
+    <module>spring-cloud-gcp-starter-sql-mysql</module>
+    <module>spring-cloud-gcp-starter-sql-postgresql</module>
+    <module>spring-cloud-gcp-starter-sql-mysql-r2dbc</module>
+    <module>spring-cloud-gcp-starter-sql-postgres-r2dbc</module>
+    <module>spring-cloud-gcp-starter-security-iap</module>
+    <module>spring-cloud-gcp-starter-vision</module>
+    <module>spring-cloud-gcp-starter-data-firestore</module>
+    <module>spring-cloud-gcp-starter-bigquery</module>
+    <module>spring-cloud-gcp-starter-security-firebase</module>
+    <module>spring-cloud-gcp-starter-secretmanager</module>
+    <module>spring-cloud-gcp-starter-kms</module>
+
+    <!--  preview modules  -->
```

Comment:
So, is the list of generated modules going here or into `spring-cloud-previews/pom.xml`?

Reply:
Yes, the generated modules will be added here. example [here](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/draft-generated/spring-cloud-gcp-starters/pom.xml#L47)

Reply:
Ah, got your point. I somehow did not wrap my head around the current parent/aggregator structure in `spring-cloud-gcp-starters/pom.xml`. Made some updates, now:

`spring-cloud-previews/pom.xml` is the aggregator pom that will have all generated modules listed. 
- it is used as part of the generation process, to run formatter on all generated modules. This is to get rid of un-used classes in imports, which gapic-generator does not have the capability to do itself. 
- added to `spring-cloud-gcp/pom` `default` profile, to include these modules in build from project root. (when ready to release, add to `release` profile then.)

`spring-cloud-gcp-starters/pom.xml` only serves as parent pom for these modules, modules will not be added to it.

You can see chagnes [here](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1394/commits/74c96dc7305e416a670d9bfdcb6e8839188c9d99). What do you think?

Reply:
Looks good.